blockstore/src/block_stores/postgres/postgres_transaction.rs

```diff
@@ -38,7 +38,7 @@ impl PostgresTransaction {
             prioritization_fees: value.prioritization_fees.map(|x| x as i64),
             cu_consumed: value.cu_consumed.map(|x| x as i64),
             recent_blockhash: value.recent_blockhash.clone(),
-            message: value.message.clone(),
+            message: BASE64.serialize(&value.message).expect("serialize message for database"),
```

Comment:
One idea could be avoid serialization if it isn not necassary.

Reply:
I moved that to this place. previously it was on the main mapper